oaknational/Oak-Web-Application
# Curric/curriculum unit modal iteration

## Description

- Accordion Component, test file, storybook update
- CurriculumUnitDetails component, test file
- Sidebar component updated
- UnitModal component updated
- UnitTabBanner component
- Updates SDK and GQL query to point at mv_curriculum_units_4

## Issue(s)

Fixes #CUR-269 #CUR-288

## How to test

1. Go to https://deploy-preview-1974--oak-web-application.netlify.thenational.academy/teachers/curriculum
2. Use subject/phase selector to choose a subject (Not Geography)
3. Select unit tab
4. Click unit info for any unit without unit optionality label to see modal open with corresponding unit info
5. Select any unit with optionality label, to view modal open with unit option selector with the titles of all optional units
6. Select optional unit info button to see submodal render with the unit information of the optional unit

## Screenshots

## How it used to look:
<img width="845" alt="Screenshot 2023-10-05 at 16 22 13" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/8bda0cd7-6f90-4dc7-b65e-89f9a26833f1">
<img width="382" alt="Screenshot 2023-10-05 at 16 22 04" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/1608da9d-ebe0-4d59-a823-9a20b0cc7642">
<img width="766" alt="Screenshot 2023-10-05 at 16 23 20" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/6055882c-3a4a-47d0-a9ba-45883cfd7a0f">



## How it should now look:
<img width="946" alt="Screenshot 2023-10-05 at 16 22 36" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/4f77cc3f-7ecc-42fb-b103-07a112cd28da">
<img width="865" alt="Screenshot 2023-10-05 at 16 18 40" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/09724d90-5276-44d1-971d-2cc9c3de027a">
<img width="758" alt="Screenshot 2023-10-06 at 09 50 10" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/c3b04639-16a9-4d14-ab30-c087ec8ed419">



## Checklist
